src/test/java/uk/gov/justice/hmpps/prison/service/OffenderDatesServiceTest.java

```diff
@@ -381,6 +384,21 @@ private static void assertOffenderCalculatedKeyDates(OffenderCalculatedKeyDates
             .build());
     }
 
+    @Test
+    void findLatestTusedDataIsReturnedCorrectly() {
+        LatestTusedData latestTusedData = new LatestTusedData(LocalDate.of(2023, 1, 3), null, null, "A1234AA");
+        when(offenderBookingRepository.findLatestTusedDataFromNomsId(anyString())).thenReturn(Optional.of(latestTusedData));
+        LatestTusedData returned = service.getLatestTusedDataFromNomsId("A1234AA");
+        assertEquals("A1234AA", returned.getOffenderNo());
```

Comment:
should be using `assertThat` rather than `assertEquals`, I tried to remove all of them a while ago but looks like some have snuck back in

Reply:
I'll sort them when I address mine